### DISCUSSION TITLE
train NuClick model from scratch

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1095] [Q&A]
Hi!
The pathology doc [here](https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/pathology) states, "for **NuClick**, training task for monailabel is not yet supported". So for now, we can't train a NuClick model from scratch like DeepEdit, correct?

**Comment by SachidanandAlle** [comment]
In latest version we are incrementally adding the feature support.. in latest version, you can fine tune your nuclick model.. but with limited options that you see in qupath..

**Reply by hw-ju** [reply]
Thanks @SachidanandAlle! Can we train from scratch or fine tune nuclick model in DSA?
Add @alex-virodov for visibility.

**Reply by SachidanandAlle** [reply]
You can train from scratch.. or fine tune.. you can discard pretrained model if you want to train from scratch
